BUAS Ray Tracer/raytracer.cpp

```diff
@@ -39,6 +46,8 @@ namespace brt
 	{
 		float lastelapsedtime = 0.f;
 
+		usefulFunction();
+
```

Comment:
pretty neat

Reply:
https://github.com/udacity/deep-learning-v2-pytorch/blob/master/intro-to-pytorch/Part%203%20-%20Training%20Neural%20Networks%20(Solution).ipynb

Reply:
lol